crates/rooch-rpc-client/src/wallet_context.rs

```diff
@@ -27,18 +27,19 @@ use std::sync::Arc;
 use std::time::Duration;
 use tokio::sync::RwLock;
 
-pub struct WalletContext {
+pub struct WalletContext<K: Ord, V> {
     client: Arc<RwLock<Option<Client>>>,
-    pub client_config: PersistedConfig<ClientConfig<RoochAddress, RoochKeyPair>>,
+    pub client_config: PersistedConfig<ClientConfig>,
     pub server_config: PersistedConfig<ServerConfig>,
+    pub keystore: Keystore<K, V>,
```

Comment:
We have not found a suitable approach to supporting the Ethereum transaction in CLI. We can make the `Keystore<K<V>` to `Keystore<RoochAddress, RoochKeyPair>`. After we support other KeyPairType to the WalletContext, we can refactor it later.

Reply:
I merge this PR first, we can refactor it later.